pkg/services/publicdashboards/service/service.go

```diff
@@ -449,3 +407,83 @@ func GenerateAccessToken() (string, error) {
 	}
 	return fmt.Sprintf("%x", token[:]), nil
 }
+
+func (pd *PublicDashboardServiceImpl) getCreatePublicDashboard(ctx context.Context, dto *SavePublicDashboardDTO) (*PublicDashboard, error) {
```

Comment:
Those verbs at the beginning sound weird, this function creates an entity or instance of a public dashboard from a DTO to send to the store layer. Maybe something like `newPublicDashboard`, `createPublicDashboard`, `createPublicDashboardInstance`, `transformDTO`. And then, update the `getUpdate` method as well. I know naming is hard 😅

Reply:
mm yes, I like those examples. I'd go for `newPublicDashboard`, but I'm thinking about the update naming.
Maybe to be consistent, use `createPublicDashboardInstance` and `updatePublicDashboardInstance`. The problem here is that it sound like it's making a persistence action.

Reply:
finally, we agreed to use `newCreatePublicDashboard` and `newUpdatePublicDashboard`